docs/sources/shared/refs-example.md

```diff
@@ -0,0 +1,48 @@
+---
+date: 2024-03-18
+description: An example of the `refs` front matter.
+title: An example of the `refs` front matter
+---
+
+A partial front matter containing the `refs` field looks like:
+
+```yaml
+refs:
+  <KEY>:
+    - pattern: <URL PATH PREFIX>
+      destination: <FULL URL>
+    - pattern: <URL PATH PREFIX>
+      destination: <FULL URL>
+```
+
+Hugo checks each `pattern` and `destination` pair, in order, against the page's URL.
+If the page's URL matches the pattern _`URL PATH PREFIX`_, then Hugo uses `destination` as the link destination.
+If no pattern matches the current page, Hugo logs a build error.
```

Comment:
Can this be less of an explanation of Hugo behaviour and more a piece of direction. I've left in a bit of tht content, but I wonder if we can even just take that out. Maybe something like (but obviously this isn't exactly right)?
```suggestion

- _`URL PATH PREFIX`_ - Enter the part of the page URL that represents the project in which these docs will appear. 
- `destination` - Enter the full URL of the destination page for that project including trailing slashes.

If the page's URL matches the pattern _`URL PATH PREFIX`_, then Hugo uses `destination` as the link destination.
If no pattern matches the current page, Hugo logs a build error.
```